kstoimenov/llvm-project
# [LSAN][NFC] Add a new line to a log

- [LLVM][OpenMPOpt] Fix a crash when associated function is nullptr (#66274)
- [analyzer] RetainRelease: Extend the workaround for VTCompressionSessionEncode{,MultiImage}Frame.
- [Coroutine][DebugInfo] Remove the memory attributes on coro-async-declaration.ll (NFC) (#66088)
- [ELF][test] Test the order of PPC64 .got, .toc, and .branch_lt
- [ELF][PPC64] Use the regular placement for .branch_lt
- [ARM][ISel] Fix crash of ISD::FMINNUM/FMAXNUM (#65849)
- [JITLink][PowerPC] Fix incorrect assertion of addend for R_PPC64_REL24
- [lldb] Support Unicode in the prompt (#66312)
- [C++20] [Modules] [Serialization] Check input file contents with option ForceCheckCXX20ModulesInputFiles enabled
- [lldb] Fix PExpectTest when using utf-8 as the encoding
- [gn build] Port 45c160510f6c
- [flang][openacc] Fix unstructured code in OpenACC region ops (#66284)
- [flang][openacc] Support labeled DO loop after acc loop directive (#66294)
- [flang][openacc] Support labeled DO loop after acc combined directive (#66296)
- [lldb] Add a setting to customize the prompt color (#66218)
- Revert "Revert "Revert "[clang] Add experimental option to omit the RTTI component from the vtable when -fno-rtti is used"""
- [AArch64][CodeGen] Fix wrong operand order when creating vcmla intrinsic (#65278)
- [lldb] Improve comments in Editline.h (NFC)
- Fix PR description HTML escaping, the surrounding `<pre>` tag shouldn't be escaped (#66322)
- [libc] Remove common_libc_tuners.cmake and move options into config.json. (#66226)
- [lldb] Simplify color logic in (IOHandler)Editline (NFC)
- [flang][openacc] Use OpenACC terminator instead of fir.unreachable after Stop stmt (#66325)
- Remove CODEOWNERS file and copy missing paths into new-prs-labeler.yml (#66145)
- [libc][NFC] Make the dummy header target under overlay build a library. (#66329)
- workflows/pr-subscriber: Use our own custom concurrency implementation (#66263)
- [AMDGPU] Fix `HasFP32Denormals` check in FDIV32 lowering (#66212)
- [clang] Bail out when handling union access with virtual inheritance
- [clang-format] Fix a bug in annotating `&&` followed by `*` or `&` (#65933)
- [InstCombine] Propagate nsw flag when negating
- [clang] Include `expected-no-diagnostics` in newly-added test (NFC)
- [clang][Interp] Actually consider ConstantExpr result
- [mlir][ArmSME] Lower vector.outerproduct to FMOPA/BFMOPA (#65621)
- [RISCV] Update Zicntr and Zihpm to version 2p0 (#66323)
- [LoongArch] Support llvm.is.fpclass for f32 and f64
- [mlir][bufferization] Empty tensor elimination based on SubsetOpInterface (#65766)
- [lldb] Format Python files in scripts and utils (#66053)
- [clang] Don't inherit dllimport/dllexport to exclude_from_explicit_instantiation members during explicit instantiation (#65961)
- [libc][utils] cpp::always_false to enable static_assert(false) (#66209)
- [mlir] Allow loop-like operations in `AbstractDenseForwardDataFlowAnalysis` (#66179)
- [AArch64] Add LRINT/LLRINT/LROUND/LLROUND FP16 lowering without fullfp16 (#66174)
- Add GTEST_ALLOW_UNINSTANTIATED_PARAMETERIZED_TEST to gwp_asan unittests
- [clang][bpf] Fix invalid RUN lines in stack protector warning test (#65251)
- [NFC] Minimize header includes (#66339)
- [clang][dataflow] In tests, error out if we didn't find any matching target functions. (#66197)
- [OpenMP][OMPIRBuilder] OpenMPIRBuilder support for requires directive
- [OpenMP][Flang][MLIR] Lowering of OpenMP requires directive from parse tree to MLIR
- [OpenMP][Flang][MLIR] Lowering of requires directive from MLIR to LLVM IR
- [flang/mlir] Fix "file too big" build error on CYGWIN.
- [analyzer] Fix StdLibraryFunctionsChecker crash on surprising sink node (#66109)
- [analyzer] Fix stdin declaration in C++ tests (#66074)
- [analyzer] Make socket `accept()` propagate taint (#66074)
- [analyzer] Propagate taint for wchar variants of some APIs (#66074)
- [AArch64][Clang] Implement ACLE rintn intrinsics (#66112)
- [mlir][bufferization] Add an ownership based buffer deallocation pass (#66337)
- [lldb][Docs] Link to testing page from debugging page
- Fixup "[analyzer] Propagate taint for wchar variants of some APIs"
- Reapply "Revert "[MemCpyOpt] implement multi BB stack-move optimization"
- [mlir][memref] Fix MemrefToLLVM lowering pattern for memref.transpose
- [mlir][CRunnerUtils] Fix iterators accessing MemRefs with non-zero offset
- Turn off memory leaks in unit test
- [mlir] [memref] Fix alignment bug in memref.copy lowering
- [llvm][mlir][bazel] Fix build.
- [clang][Interp][NFC] Call delegate() for the RHS of a comma op
- [mlir][bufferization][NFC] Introduce BufferDeallocationOpInterface (#66349)
- [SVE] Ensure SVE call operands passed via memory are correctly initialised. (#66070)
- [github] Label PRs touching CMake-related files with the 'cmake' tag (#66247)
- [SimplifyCFG] handle monotonic wrapped case for D150943 (#65882)
- [mlir][bufferization] BufferDeallocationOpInterface: support custom ownership update logic (#66350)
- [clang-format] More on unbreakable strings in TypeScript (#66321)
- [libc] Fix some warnings (#66366)
- [mlir][bufferization] Fix BAZEL build
- [mlir][transform] Fix crash when op is erased during transform.foreach (#66357)
- [libc] Fix start up crash on 32 bit systems (#66210)
- [lit][NFC] Remove stray character in docstring
- [mlir][Linalg] NFC - Simplify transform script
- [IR] Enable load/store/alloca for arrays of scalable vectors.
- [lit] Fix some issues from --per-test-coverage (#65242)
- [mlir][bufferization] Implement BufferDeallocationopInterface for scf.forall.in_parallel (#66351)
- [mlir][transform] Check for invalidated iterators on payload IR mappings (#66369)
- [LICM] Simplify isLoadInvariantInLoop given opaque pointers (#65597)
- [flang] Do not finalize main program variables. (#66326)
- [flang] Select proper library APIs for derived type io. (#66327)
- [AArch64] New subtarget features to control ldp and stp formation (#66098)
- Resign as code owner of branch weights and block frequency
- clang/AMDGPU: Fix accidental behavior change for __builtin_amdgcn_ldexph (#66340)
- [LSAN][NFC] Add a new line to a log
